packages/db/test/user-storage.spec.js

```diff
@@ -113,17 +114,42 @@ describe('Users used storage', () => {
       tag: 'StorageLimitBytes',
       value: '2199023255552'
     })
-    const cid = 'bafybeibvuy3vcepqxy4plr34twv22vvxol2jjhmjxcrcvuhea5226whpsm'
+    cid = await randomCid()
     await createUpload(dbClient, Number(user4._id), Number(authKey4), cid, {
       dagSize: largeFileSize
     })
 
+    user5 = await createUser(dbClient, {
+      name: 'test5 restricted',
+      email: 'test5@email.com'
+    })
+    authKey5 = await createUserAuthKey(dbClient, Number(user5._id), {
+      name: 'test5-key'
+    })
+    await dbClient.createUserTag(Number(user5._id), {
+      tag: 'StorageLimitBytes',
+      value: '2199023255552'
+    })
+    await dbClient.createUserTag(Number(user5._id), {
+      tag: 'HasAccountRestriction',
+      value: 'true'
+    })
+    await dbClient.createUserTag(Number(user5._id), {
+      tag: 'HasPsaAccess',
+      value: 'true'
+    })
+    cid = await randomCid()
+    await createUpload(dbClient, Number(user5._id), Number(authKey5), cid, {
+      dagSize: largeFileSize
+    })
+
     // User               | upload  | pinned  | quota   | percentage of quota
     // ----------------------------------------------------------------------
     // test1@email.com    | 1       | 2       | 1TiB    | < 75%
     // test2@email.com    | 2       | 0       | 1TiB    | 80%
     // test3@email.com    | 2       | 1       | 1TiB    | 90%
     // test4@email.com    | 1 (XL)  | 0       | 2TiB    | > 90%
+    // test5@email.com    | 1 (XL)  | 0       | 2TiB    | ACCOUNT RESTRICTED
```

Comment:
Should there be a test added for the expected result of `getUsersByStorageUsed` with respect to user5?

Reply:
Good point, have added an explicit test.